pkg/controller/gslb/gslb_controller_test.go

```diff
@@ -626,6 +626,16 @@ func TestGslbController(t *testing.T) {
 		}
 	})
 
+	t.Run("Reflect GeoTag in the Status as unset by default", func(t *testing.T) {
```

Comment:
Not sure if this should be here. Test itself is needed for sure  but on controller level we should check reconciliation context. This should be in depresolver tests as it is just testing default GeoTag value when no input given.

Reply:
@kuritka it is testing exactly subtle bug. Not just the config construction but also the Gslb.Status propagation with default value. The place where we hit the API errors during latest deployments.

Reply:
ok, so than I have nothing against...